contract/src/error.rs

```diff
@@ -142,6 +142,19 @@ pub enum NFTCoreError {
     InvalidAccessKeyName = 137,
     InvalidCheckForUpgrade = 138,
     InvalidNamedKeyConvention = 139,
+    InvalidTokenEvent = 140,
+    MissingTokenEventId = 141,
+    InvalidTokenEventId = 142,
+    InvalidTokenEventOrder = 143,
+    FailedToRecordMintEvent = 144,
+    FailedToRecordBurnedEvent = 145,
+    FailedToRecordApproveEvent = 146,
+    FailedToRecordTransferEvent = 147,
+    FailedToRecordMetadataUpdateEvent = 148,
+    InvalidStartingEventId = 149,
+    InvalidLastEventId = 150,
+    MissingEventsMode = 151,
+    InvalidEventsMode = 152,
```

Comment:
Error excepted 
```
MissingTokenEventId 
InvalidTokenEventId
InvalidTokenEventOrder
FailedToRecordMintEvent
FailedToRecordApproveEvent
FailedToRecordTransferEvent
InvalidStartingEventId
InvalidLastEventId
```

do not seem to be used

```suggestion
    InvalidTokenEvent = 140,
    MissingEventsMode = 141,
    InvalidEventsMode = 142,
```

Reply:
Removed unused errors, I expect they were introduced in cep78 and have now become unnecessary